crates/rust-analyzer/src/config.rs

```diff
@@ -171,9 +173,9 @@ config_data! {
         /// ```
         /// .
         checkOnSave_overrideCommand: Option<Vec<String>> = "null",
-        /// Check for a specific target. Defaults to
+        /// Check for specific targets. Defaults to
         /// `#rust-analyzer.cargo.target#`.
-        checkOnSave_target: Option<String>               = "null",
+        checkOnSave_target: Vec<String>                  = "[]",
```

Comment:
We should still accept non arrays as well, easiest would probably be to use a `Vec<String>` wrapper struct here and use the `single_or_array` function for deserialization thats already defined in the module

Reply:
Fixed